Documentation/ceph-osd-mgmt.md

```diff
@@ -63,23 +60,51 @@ through the removal process:
 If all the PGs are `active+clean` and there are no warnings about being low on space, this means the data is fully replicated
 and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly clean and you will need to proceed anyway.
 
-### From the Toolbox
+### Prevent the rook operator to re-create the OSD pod
+
+#### Host-based cluster
+
+Update your CephCluster CR. Depending on your CR settings, you may need to remove the device from the list or update the device filter.
+If you are using `useAllDevices: true`, no change to the CR is necessary.
+
+#### PVC-based cluster
+
+1. Shrink the number of OSDs in the `storageClassDeviceSet` in the CephCluster CR.
+   - `kubectl -n rook-ceph edit cephcluster rook-ceph`
```

Comment:
Can we suggest a `patch` command instead to make it easier for users?

Reply:
Added.